code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -33,7 +33,7 @@
 	icon_state = "trashbag"
 	item_state = "trashbag"
 
-	w_class = 4
+	w_class = 3

```

Comment:
I can definitely agree with this statement. Trash bags, and TBoH are toolbelt+ and bluspacebelt+.